sdk/inc/azure/core/internal/az_mqtt5_topic_parser_internal.h

```diff
@@ -81,6 +81,19 @@
  */
 #define _az_MQTT5_TOPIC_PARSER_COMMAND_ID_HASH 2624200456
```

Comment:
This should match `_az_MQTT5_TOPIC_PARSER_NAME_KEY` ? (i.e. `_az_MQTT5_TOPIC_PARSER_NAME_HASH`)